package.json

```diff
@@ -36,7 +36,8 @@
     "unit": "jest --coverage",
     "test": "yarn lint && yarn unit",
     "test:integration": "yarn lint && jest --config=jest-integration.config.js",
-    "watch": "jest --watch",
+    "watch": "yarn build && cross-env NODE_ENV=development node --watch ./dist ",
```

Comment:
This only watches for changes in the compiled files. A `nodemon` setup with `ts-node` is a solution to it.